package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "streetmerchant",
+  "version": "3.4.1",
```

Comment:
Adding a version number here will increase the maintenance burden because it requires updating versions in two places. I can change `banner.ts` to use this number if that's preferable. 

The changes in this file aren't strictly necessary as it's easy to make them myself and then tell `git` to make them `assume-unchanged`.

Reply:
Before we make this change, I'd like to update the pipeline to manage this for us. This is easy enough, but just needs an update.